ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.16-nightly-x86.yaml

```diff
@@ -44,21 +45,20 @@ tests:
       COMPUTE_NODE_REPLICAS: "3"
       COMPUTE_NODE_TYPE: Standard_D4s_v5
       CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v5
-      CUSTOM_AZURE_REGION: centralus
       PROFILE_TYPE: reporting
     test:
     - chain: openshift-qe-control-plane
-    workflow: openshift-qe-installer-azure-ipi-ovn-ipsec
+    workflow: openshift-qe-installer-azure
```

Comment:
@qiliRedHat: Please update the job name, as the workflow has changed. Else please revert this.